README.md

```diff
@@ -27,6 +27,20 @@ that creates additional scrape targets. Each scrape target is expected
 to expose a `/metrics` HTTP path that exposes its metrics in a
 Prometheus compatible format.
 
+For example if it desired to scrape metrics regarding the state of
+objects in a Kubernetes cluster, the
+[kube-state-metrics](https://charmhub.io/kube-state-metrics) charm may
+be used for this purpose. All that is required, in addition to
+deploying Prometheus as above, is to deploy the `kube-state-metrics`
+charm and add a relation with the Prometheus charm as shown below.
+
+
+    juju deploy ch:kube-state-metrics
```

Comment:
`cs:`?

Reply:
I think charmhub is `ch`.

Reply:
I only see `cs` (as in charm store?) in the output of `juju deploy --help | grep -E 'cs:|ch:'`

Reply:
let me ask/check.